app/models/salt_handler/minion_orchestration.rb

```diff
@@ -11,17 +11,26 @@ def self.can_handle_event?(event)
     return false unless event.tag =~ TAG_MATCHER
     parsed_event_data = JSON.parse event.data
     parsed_event_data["fun"] == "runner.state.orchestrate" &&
-      (parsed_event_data["fun_args"].first == "orch.kubernetes" ||
-       parsed_event_data["fun_args"].first["mods"] == "orch.kubernetes")
+      salt_fun_args_match(parsed_event_data, "orch.kubernetes", "orch.update")
+  end
+
+  def self.salt_fun_args_match(parsed_event_data, *orchestrations)
+    fun_args = parsed_event_data["fun_args"]
+
+    orchestrations.each do |o|
+      return true if fun_args.first == o || fun_args.first["mods"] == o
+    end
+
+    false
```

Comment:
This whole loop and `return true/false` can be replaced with a one liner: `orchestrations.any? { |o| fun_args.first == o || fun_args.first["mods"] == o }`